diff --git a/node_modules/@sitecore-content-sdk/nextjs/dist/cjs/editing/utils.js b/node_modules/@sitecore-content-sdk/nextjs/dist/cjs/editing/utils.js
index f33d799..9700ce5 100644
--- a/node_modules/@sitecore-content-sdk/nextjs/dist/cjs/editing/utils.js
+++ b/node_modules/@sitecore-content-sdk/nextjs/dist/cjs/editing/utils.js
@@ -196,6 +196,7 @@ const getEditingRequestHtml = async (requestUrl, propagatedQsParams, propagatedH
     const pageRes = await dataFetcher
         .get(requestUrl.toString(), {
         credentials: 'include',
+        cache: 'no-store',
         headers: propagatedHeaders,
     })
         .catch((err) => {
diff --git a/node_modules/@sitecore-content-sdk/nextjs/dist/esm/editing/utils.js b/node_modules/@sitecore-content-sdk/nextjs/dist/esm/editing/utils.js
index 7facfd8..266d5a2 100644
--- a/node_modules/@sitecore-content-sdk/nextjs/dist/esm/editing/utils.js
+++ b/node_modules/@sitecore-content-sdk/nextjs/dist/esm/editing/utils.js
@@ -185,6 +185,7 @@ export const getEditingRequestHtml = async (requestUrl, propagatedQsParams, prop
     const pageRes = await dataFetcher
         .get(requestUrl.toString(), {
         credentials: 'include',
+        cache: 'no-store',
         headers: propagatedHeaders,
     })
         .catch((err) => {
